### PR TITLE
Moving firebase api key to config file

### PR DESCRIPTION
**Pull request by pmosse**

### CODE REVIEW

**Comment by lautarodragan** [comment]
I think they key was duplicated and hardcoded somewhere else too, did you check that?

**Comment by pmosse** [comment]
@lautarodragan You are right, the server was being started from a second location

File: node/src/notifications/appserver.ts [hunk 1]
```diff
@@ -164,5 +173,5 @@ function createServer (serverKey: string, port: number) {
 export default createServer
 
 if (!module.parent) {
-  createServer(SERVER_API_KEY, 5500)
+  createServer(configuration.serverApiKey, 5500)
```
Comment (lautarodragan) [inline node/src/notifications/appserver.ts]: I believe this code is actually never run (and would be a duplicate of the code placed in `node/src/bin/notifications.ts`). Can you confirm this and if so, delete the unneeded code?

**Comment by pmosse** [comment]
@lautarodragan You are right, it was not necessary. I am getting a connection refused error when it starts and tries to connect to the DDBB, but it should be something with my env, I was not getting this before.

File: node/src/notifications/appserver.ts [hunk 2]
```diff
@@ -10,8 +10,17 @@ const fcm = require('fcm-node')
...
-const SERVER_API_KEY='56a75cc15cc842384321b5fe614d54d4238049c49dec61feaeed79968156a821'
+const command = process.argv[2]
+const commandArgument = process.argv[3]
+
+if ((command !== '--configuration' && command !== '-c') || !commandArgument) {
+  console.error('Usage: [--configuration <path>] [-c <path>]')
+  process.exit()
+}
+
+const configuration = getConfiguration(commandArgument)
```
Comment (lautarodragan) [inline node/src/notifications/appserver.ts]: Now that you removed the unused code, this is no longer necessary. You should be able to remove all this duplicated code entirely.

**Comment by pmosse** [comment]
@lautarodragan Done! Should I deploy this to testing and test it there too?

File: node/src/notifications/appserver.ts [hunk 3]
```diff
@@ -161,8 +157,4 @@ function createServer (serverKey: string, port: number) {
   })
 }
 
-export default createServer
-
-if (!module.parent) {
-  createServer(SERVER_API_KEY, 5500)
-}
+export default createServer
```
Comment (lautarodragan) [inline node/src/notifications/appserver.ts]: Let's refactor this to avoid using default exports while you're at it.

**Comment by pmosse** [comment]
@lautarodragan Is this one ready to be merged?

**Comment by lautarodragan** [comment]
@pmosse yup

**Pull request merged**